paper/sections/materials.tex

```diff
@@ -29,16 +51,26 @@ \subsection{Dataset}
 percentage of all standard amino acids present within a protein, as defined by~\cite{gromiha2010protein}. Also known
 as Monopeptide Composition (MPC).
 \item \textbf{Dipeptide Composition (DPC)}: a feature vector of $400$ values ranging from $0$ -- $100$ indicating the
-percentage of all possible ordered dipeptide pairs present within a protein, as defined by~\cite{gromiha2010protein}.
-\item \textbf{Physico-Chemical Composition (PHC)}: a feature vector of $11$ present/absent binary values corresponding
-to physico-chemical residue classes, including: Aliphatic, Neutral, Aromatic, Hydrophobic, Charged, Positively charged,
+percentage of all possible amino acid pairs present within a protein, as defined by~\cite{gromiha2010protein}.
```

Comment:
```suggestion
percentage of all possible ordered amino acid pairs present within a protein, as defined by~\cite{gromiha2010protein}.
```

The fact that these pairs are ordered is important, as it changes the length from 20c2 (190) to 20^2 (400)

Reply:
So, if there are 400 features, then that mean, it was not ordered? because there are 400 numerical values for each sequence in this regard.

Reply:
400 values means that it **is** ordered: position 1 (20) x position 2 (20) = 400. If order didn't matter, then it would just be 20c2 = 20! / (18!) (2!) = 20*19/2 = 190.

Reply:
Ok 👍